### ISSUE TITLE
fix(secure-coding): no-directive-injection flags the correct DOMPurify call

### ISSUE THREAD

**Issue by ofri-peretz** [ofri-peretz/eslint#515]
The rule reports the recommended safe pattern:

```js
DOMPurify.sanitize(html, { ALLOWED_TAGS, ALLOWED_ATTR })
```

This is the configured, allow-listed sanitizer call — the answer the rule should be steering people toward. Reporting it tells a developer who did the right thing that they did the wrong thing.

Carried over from the FP worklist in `DISTRIBUTION-PLAN.md`; not yet reproduced in isolation.

## Acceptance

- TDD: the call above as `valid`, failing first
- A genuinely unsafe directive (`sanitize(html, { ADD_TAGS: ['script'] })`, or an unsanitized assignment) still reports
